app/src/main/res/xml/quran_preferences.xml

```diff
@@ -147,29 +147,16 @@
   </PreferenceCategory>
 
   <PreferenceCategory
```

Comment:
i think @ozbek was saying, since `PreferenceCategory extends Preference`, instead of having the `PreferenceCategory` have a nested `Preference` for advanced settings, can you just make `PreferenceCategory` handle advanced settings without adding a child in xml?

i.e. instead of what you have now:

```xml
<PreferenceCategory android:key="">
   <Preference android:key="">
        <intent android:targetPackage="" />
   </Preference>
</PreferenceCategory>
```

instead, would it work if you move everything on the `Preference` up to the `PreferenceCategory` instead and remove the preference? i.e.

```xml
<PreferenceCategory android:key="">
    <intent android:targetPackage="" />
</PreferenceCategory>
```